{"name": "故障告警数量-传入错误的参数值", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 != 0", "trace": "login_start = {'Authorization': 'Bearer eyJhbGciOiJub25lIn0.eyJ1c2VyIjp7ImlkIjoxLCJ1c2VybmFtZSI6ImFkbWluIiwiZW5hYmxlIjoxLCJtb2JpbGUi...6MzIiLCJsZXZlbCI6MSwid2ViTWVudUxpc3QiOm51bGwsImFwcE1lbnVMaXN0IjpudWxsLCJjcmVhdGVVc2VySWQiOjF9XX0sInN1YiI6ImFkbWluIn0.'}\n\n    @allure.feature(\"运维管理平台\")\n    @allure.story(\"首页\")\n    @allure.title(\"故障告警数量-传入错误的参数值\")\n    @allure.description(\"\")\n    def test_deviceError_count_003(login_start):\n            Authorization =login_start\n            method=data[\"deviceError_count\"][2][\"Method\"]\n            apiurl=data[\"deviceError_count\"][2][\"ApiUrl\"]\n            headers=data[\"deviceError_count\"][2][\"Headers\"]\n            params = data[\"deviceError_count\"][2][\"Params\"]\n            #往请求头里面添加Authorization\n            headers.update(Authorization)\n            #发送请求\n            response=send_requests(method,apiurl,headers=headers,params=params)\n            #断言\n>           assert response['code'] != 0\nE           assert 0 != 0\n\nopsPC\\test_homePage.py:299: AssertionError"}, "attachments": [{"name": "log", "source": "1b3f7d0a-5402-4ff4-8480-3f46d3c3b18c-attachment.txt", "type": "text/plain"}], "start": 1690421482102, "stop": 1690421482530, "uuid": "ac961368-6252-4a91-b558-2a0cfad51de5", "historyId": "57ffa21a8bf27314c984bbe2ae126ca3", "testCaseId": "359e5974034b84dc0cb8e920abeda087", "fullName": "opsPC.test_homePage#test_deviceError_count_003", "labels": [{"name": "story", "value": "首页"}, {"name": "feature", "value": "运维管理平台"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_homePage"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "6556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_homePage"}]}